libs/net/dal/Migrations/Initial/Up/PostUp/02-DataSourceSchedule.sql

```diff
@@ -52,4 +53,28 @@ INSERT INTO public.data_source_schedule (
 );
 END IF;
 
+IF (CASTANET_ID IS NOT NULL) THEN
+  INSERT INTO public.data_source_schedule (
+    "data_source_id"
+    , "schedule_id"
+    , "created_by_id"
+    , "created_by"
+    , "created_on"
+    , "updated_by_id"
+    , "updated_by"
+    , "updated_on"
+    , "version"
+  ) VALUES (
+    CASTANET_ID  -- data_source_id
+    , 3  -- schedule_id
```

Comment:
I was fairly certain that this was a syndication data source and the schedule is continuous.  You have updated the schedule type to be `1`, why have you entered `3` here?

Reply:
Looks like I misinterpreted this.  You don't need to do anything.